Source/NSView.m

```diff
@@ -5165,277 +5166,8 @@ - (void) setUserInterfaceLayoutDirection: (NSUserInterfaceLayoutDirection)dir
   return;
 }
 
-/**
```

Comment:
This looks like an error. You seem to have removed all the layout code that was added last year.